qemu/tests/cfg/trim_support_test.cfg

```
@@ -20,3 +20,17 @@
     drv_extra_params_stg1 = "discard=unmap"
     host_check_cmd = "du --block-size=1 %s/${stg_name}.qcow2"
     guest_trim_cmd = "defrag.exe %s: /l /u /v"
+    variants:
+         - @default:
+         - with_4k_disk:
+             only virtio_blk
+             block_size = 4K
+             physical_block_size_stg1 = 4096
+             logical_block_size_stg1 = 4096
+             guest_trim_cmd = 'powershell "Optimize-Volume -DriveLetter %s -ReTrim -Verbose"'
+             event_id = 55
+             searched_keywards = "A corruption was discovered"
```

typo: searched_keywords

---

```suggestion
             searched_keywords = "A corruption was discovered in the file system structure on volume"
```
Better to add the scope of storage info. like the `volume` keyword

---

Many thanks @YongxueHong ,
Test with the old driver ,and will paste the lastest result with lastest driver later
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.trim_support_test.with_4k_disk.q35: FAIL: Disk corruption after trim for 4K (279.07 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-03-22T02.05-56b0c07/results.html